File: examples/3_usando_modulos/009_manipulando_texto/ex024_cidade_comeca_com_santo/ex024.py
# faça um programa que leia o nome de uma cidade e diga se ela começa ou não com "Santo" 

city = input('Digite o nome de uma cidade: ')

if city.isnumeric():

    print('Informe um valor válido!')

else:

    if "Santo" in city and city[0] == "S":

        print(f'Sim, a cidade começa com "Santo". \nA cidade digitada foi {city}!')

    else:

        print(f'Não, a cidade digitada foi {city}!')

File: examples/3_usando_modulos/009_manipulando_texto/ex024_cidade_comeca_com_santo/ex024.js
function isNumber(variable) {

    if (variable.match(/[0-9]/)) {
        return true;
    } else {
        return false;
    }
}

function haveSanto(variable) {

    if (variable.indexOf('Santo') === 0 || variable.indexOf('santo') === 0) {
        return true;
    } else {
        return false;
    }

}

function confirmCity() {

    if (isNumber(inputCity.value) || inputCity.length == "") {
        
        divAnswer.innerHTML = `<div class="answer error">Informe uma cidade válida!</div>`;

    } else {

        if (haveSanto(inputCity.value)) {

            divAnswer.innerHTML = `<div class="answer"></div>`;
            divAnswer.innerHTML += `<div class="answer">Sim, a cidade ${inputCity.value} tem Santo no nome e começa com a mesma!</div>`;

        } else {

            divAnswer.innerHTML = `<div class="answer"></div>`;
            divAnswer.innerHTML += `<div class="answer">Não, a cidade informada foi a cidade de ${inputCity.value}!</div>`;
        }
    }
}

let inputCity = document.querySelector('#input1');
let divAnswer = document.querySelector('.answer');